src/probnum/filtsmooth/gaussfiltsmooth/kalman.py

```diff
@@ -46,9 +47,62 @@ def iterated_filtsmooth(
         Parameters
         ----------
         regression_problem
+        init_posterior
+            Initial posterior to linearize at. Defaults to computing a (non-iterated)
+            smoothing posterior, which amounts to linearizing at the prediction
+            random variable.
         stopcrit: StoppingCriterion
             A stopping criterion for iterated filtering.
 
+        Returns
+        -------
+        SmoothingPosterior
+
+        See Also
+        --------
+        RegressionProblem: a regression problem data class
+        """
+
+        smoothing_post = init_posterior
+        info_dicts = None
+        for smoothing_post, info_dicts in self.iterated_filtsmooth_generator(
+            regression_problem, smoothing_post, stopcrit
+        ):
+            pass
+
+        if smoothing_post is None or info_dicts is None:
+            raise ValueError("Iterated filter did not perform a single iteration.")
```

Comment:
when would this valueerror happen? And would the situation you describe in the message rather be a warning than an error?

Reply:
Never. Decision: remove